.travis.yml

```diff
@@ -20,12 +20,12 @@ script:
 after_success:
   - travis-cargo doc-upload
   - if [ "${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}" != "master" ] && [ "$TRAVIS_RUST_VERSION" == "nightly" ]; then
```

Comment:
I think because it clones a branch directly, you can't see any other refs. `git fetch origin master && git checkout -b master FETCH_HEAD` might work, or even `git fetch origin master:master`.